RELEASE.md

```diff
@@ -0,0 +1,57 @@
+# Release Process
+
+Currently to release a new version of the integreatly-operator and deploy it, the following steps are required:
+
+1. Create a new branch off the base branch
+2. Execute the [pre-release script](./scripts/prepare-release.sh)
+3. Commit changes and create a PR
+4. Wait for the PR to be review and merged.
+5. Create the release tag for the base branch HEAD, and wait for the image to be available on quay.io and tag it.
+6. Create the PR against the managed-tenants repo. Stage channel first, then later on the same changes should be made to the edge and stable channels.
+
+The above process has been automated and the Jenkins pipeline can be found [here](https://master-jenkins-csb-intly.cloud.paas.psi.redhat.com/job/Delorean/job/rhmi-release/).
+
+NOTE: Red Hat VPN is required to access.
+
+# Prepare for Patch Releases
+
+In order to create patch releases, we need to make sure a release branch is created first and it is configured properly. Run the following command:
+   
+```
+delorean release openshift-ci-release --version 2.1.1-er1
+```
```

Comment:
@mikenairn do you think if we can move this to a jenkins pipeline too?

Reply:
As discussed previously i don't think we should add it to the pipeline, yet anyway. We can look at doing that later i guess, but it might require changes to the way we are triggering certain things that use "docker run".

For now I'd leave it as a step here, i would say though we should probably add it as a make target that calls the containerised delorean cli instead.

```
CONTAINER_ENGINE ?= docker

prepare-patch-release:
	$(CONTAINER_ENGINE) pull quay.io/integreatly/delorean-cli:latest:latest
	$(CONTAINER_ENGINE) run --rm quay.io/integreatly/delorean-cli:latest:latest delorean release openshift-ci-release --version 2.1.1-er1
```

wdyt?

Reply:
yeah, good idea

Reply:
I have added the make command now, mind taking another look?